### ISSUE TITLE
[Bug] [Accessibility] Shortcut doesn't work on AZERTY keyboard

### ISSUE THREAD

**Issue by YuriGaliza** [telerik/kendo-angular#4744]
### Description

On any multiselect, on a QWERTY keyboard, if you press CTRL + A, it selects all of the values in the list, but on an AZERTY keyboard, it doesn't work (it used to work on 16.3.0). . It can be reproduced on the accessibility section in Kendo page. (
https://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/keyboard-navigation)

### Steps To Reproduce

1. Use AZERTY keyboard (French speaking, so France, Belgium, Canada, ... )
2. Open any multiselect
3. Use shortcut CTRL + A
4. Nothing is selected

### Screenshots or video

<img width="1622" height="488" alt="Image" src="https://github.com/user-attachments/assets/482c1c48-76fa-48b0-81eb-ec86cce40a77" />

### Actual Behavior

1. Use AZERTY keyboard (French speaking, so France, Belgium, Canada, ... )
2. Open any multiselect
3. Use shortcut CTRL + A
4. Nothing is selected

### Expected Behavior

1. Use AZERTY keyboard (French speaking, so France, Belgium, Canada, ... )
2. Open any multiselect
3. Use shortcut CTRL + A
4. All the elements in the list are selected

### Browser

All

### Browser version

latest

### OS type

MacOS

### Last working version of the Kendo UI for Angular package (if regression).

16.3.0
